.pipelines/cni/k8s-e2e/k8s-e2e-job-template.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
             # sig-release provides test suite tarball(s) per k8s release. Just need to provide k8s version "v1.xx.xx"
             # pulling k8s version from AKS.
             eval k8sVersion="v"$( az aks show -g ${{ parameters.clusterName }}-$(make revision) -n ${{ parameters.clusterName }}-$(make revision) --query "currentKubernetesVersion")
+            echo $k8sVersion
```

Comment:
nit: Do we need this, we have the nextline as `https://dl.k8s.io/$k8sVersion)` which can tells us the used k8s version ?

Reply:
This step does not show the full curl command, only the results. The other way we could do this is change line 22 to` set -ex`.